Common/src/main/resources/META-INF/common-plugin.xml

```diff
@@ -105,7 +105,7 @@
 
     <change-notes>Tabnine is always evolving. Most updates are in the binary sidecar.</change-notes>
 
-    <idea-version since-build="193"/>
+    <idea-version since-build="193.5233.102"/>
```

Comment:
please verify (and share with us) in mixpanel that we're not ditching too many users here.
should be fine, but I think you should at least fetch some insights from mixpanel consult with your team lead or someone.

also, this can be extracted to a `gradle.properties` file I think, just so it won't repeat itself multiple times in this repo. but not critical for now.

Reply:
` 193.5662.53` is the first version of `2019.3` -> `2019.3.1`, according to https://www.jetbrains.com/idea/download/other.html. So I don't think it actually matter but not getting the  compile error